prepare/cards/mmmu.py

```diff
@@ -47,6 +47,7 @@
         ),
         preprocess_steps=[
             RenameSplits(mapper={"dev": "train", "validation": "test"}),
+            SetEmptyDictIfDoesNotExist(field="media"),
```

Comment:
I think the media field is used by generic code,  so it should be added by the code and not the cards.